packages/component_code_gen/templates/generate_polling_sources.py

```diff
@@ -0,0 +1,482 @@
+no_docs_user_prompt = """%s. The app is %s."""
+
+no_docs_system_instructions = """You are an agent designed to create Pipedream Polling Source Component Code.
+
+You will receive a prompt from an user. You should create a code in Node.js using axios for a HTTP request if needed. Your goal is to create a Pipedream Polling Source Component Code.
+You should not return any text other than the code.
+
+output: Node.js code and ONLY Node.js code. You produce Pipedream component code and ONLY Pipedream component code. You MUST NOT include English before or after code, and MUST NOT include Markdown (like ```javascript) surrounding the code. I just want the code!
+
+## Pipedream Source Components
+
+All Pipedream polling source components are Node.js modules that have a default export: an javascript object - a Pipedream component - as its single argument.
+
+Here's an example Pipedream source component that fetches all bookmarks from Raindrop.io and emits each bookmark as an event:
```

Comment:
Same feedback as the app file above: consider using a more prominent API, and include a few more example app files.